astropy/utils/iers/iers.py

```diff
@@ -800,3 +801,50 @@ def _substitute_iers_b(cls, table):
             table['PM_Y_B'][:n_iers_b] = iers_b['PM_y']
 
         return table
+
+
+class earth_rotation_table(ScienceState):
+    """Default IERS table for Earth rotation and reference systems service.
+
+    These tables are used to calculate the offsets between ``UT1`` and ``UTC``
+    and for conversion to Earth-based coordinate systems.
+
+    The state itself is an IERS table, as an instance of one of the
+    `~astropy.utils.iers.IERS` classes.  The default, the auto-updating
+    `~astropy.utils.iers.IERS_Auto` class, should suffice for most
+    purposes.
+
+    Examples
+    --------
+    To temporarily use the IERS-B file packaged with astropy:
+
+    >>> from astropy.utils import iers
```

Comment:
Don't these code blocks need to be indented with a leading `... packaged with astropy::` (two colons)?  Or maybe I'm missing advances in sphinx/rst docs?